com.borgbase.vorta.yaml

```diff
@@ -0,0 +1,89 @@
+app-id: com.borgbase.vorta
+
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+
+command: vorta
+
+finish-args:
+  # X11/Wayland
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=dri
+  # Sync
+  - --share=network
+  # Filesystem access
+  - --filesystem=home
+  - --filesystem=/run/media
+  - --filesystem=/media
+  # dconf access
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  # Tray icon
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.StatusNotifierItem-2-1
+  # Mounting
+  - --talk-name=org.freedesktop.*
+  # Allow access to keyring
+  - --talk-name=org.freedesktop.secrets
+  # SSH
+  - --socket=ssh-auth
+
+build-options:
+  env:
+    MOUNT_FUSE_PATH: ../tmp/
+
+modules:
+  - dependencies/pyqt5.json  
```

Comment:
Any reason for separating each dependency on a file itself?

Reply:
flatpak-pip-generator is quite buggy, so it was simpler to fix the bugs, module per module.
Additionally I find it more overseeable that way (and I would now have to rewrite all json files to yaml).
Any reasons not to do it that way?